task/prefetch-dependencies/0.1/prefetch-dependencies.yaml

```
@@ -62,6 +86,12 @@ spec:
         exit 0
       fi
 
+      if [ -f /mnt/config/config.yaml ]; then
+        config_flag=--config-flag /mnt/config/config.yaml
```

```suggestion
        config_flag=--config-file=/mnt/config/config.yaml
```
- typo in name
- I think the space makes this a `SOME_VARIABLE=foo some-command` situation (sets a variable and invokes a command, doesn't set a variable that contains a space)